### PR TITLE
Burnt area v7

### PR DESCRIPTION
**Pull request by erin-telfer**
latest updates include abstract text and legend changes.
I'm not sure why checks are failing

### CODE REVIEW

File: dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py [hunk 1]
```diff
@@ -201,60 +195,61 @@
...
-        "end": "1",
+        "title": "dNBR classes",
+        "begin": "-1.0",
+        "end": "1.0",
+        "ticks": ["-1.0", "-0.45", "0.1", "0.27", "0.44", "0.72", "1.0"],
         "tick_labels": {
-            "-1": {"prefix": "<"},
-            "1": {"prefix": ">"},
+            "-1.0": "-1.0",
+            "-0.45": "Unburnt",
```
Comment (pindge) [inline dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py:211]: this is wrong, need to follow the format of 

```
        "tick_labels": {
            "-1.0": {"label": "<-1"},
            "0": {"label": "0"},
            "1.0": {"label": ">1"},
        },
```

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py (hunk 1 above)
Comment (pindge) [reply dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py:211]: either `label` or `prefix`

File: dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py [hunk 2]
```diff
@@ -201,60 +195,61 @@
     "range": [-1, 1],
     "color_ramp": [
         {"value": -1.0, "color": "#0be344", },
-        {"value": 0.099, "color": "#0be344", },
-        {"value": 0.1, "color": "#f8fc11", },
-        {"value": 0.499, "color": "#f8fc11", },
-        {"value": 0.5, "color": "#f8671a", },
+        {"value": 0.0999, "color": "#0be344", },
```
Comment (pindge) [inline dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py:198]: for consistency, can we remove the `,` right after `color` definition? the ending `,` before the closing `}`

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py (hunk 1 above)
Comment (erin-telfer) [reply dev/services/wms/ows_refactored/hazards/burntarea/ows_provisional_ba_cfg.py:211]: ook, I admit that I just copied the code that Paul sent on slack. I will update to add the label component :)

**Review by pindge — APPROVED** [review]

**Pull request merged**